fava/util/date.py

```diff
@@ -94,7 +94,7 @@ def _parse_month(month):
         return months_abbr.index(month)
 
 
-def parse_date(string):
+def parse_date(string, today=None):
```

Comment:
I'm against adding unnecessary arguments just for tests. Or is there some other place where we use it?